src/external-scripts/index.ts

```diff
@@ -90,13 +100,36 @@ const sessionTracker = (
   }
 
   if (inkeepEnabled && inkeepApiKey) {
-    inkeepChatIdentifyUser(sessionState);
+    if (sessionState.user && 'uuid' in sessionState.user) {
+      inkeepChatIdentifyUser({ user: sessionState.user as InkeepUser });
+    }
   }
 };
 
-const externalScriptInjector = (externalScriptsData) => ({
+const externalScriptInjector = (
+  externalScriptsData:
+    | {
```

Comment:
Could this be extracted into a type as well? It's reused in another file in this PR